### DISCUSSION TITLE
Purchased services and its Scope 3 emissions

### DISCUSSION THREAD

**Discussion by arby1005** [USEPA/supply-chain-factors#27] [Q&A] [answered]
Regarding purchased services, what does the EEIO database genearlly cover (e.g. does it include Scope 3 emissions of the service providers)?

**Comment by WesIngwersen** [comment, marked as answer]
The Supply Chain Factors cover all products and services made/delivered in the US, so this includes services.  They are available in both the 'commodity' or 'industry' forms just like product factors and can be found in the same place and their life cycle coverage is the same.  In summary we make no distinction between products and services in the supply chain factors and the factors for any service can be used in the same way as a product factor.
